lib/zipkin-tracer/zipkin_logger_tracer.rb

```diff
@@ -1,17 +1,24 @@
 require 'zipkin-tracer/zipkin_tracer_base'
 require 'zipkin-tracer/hostname_resolver'
+require 'json'
 
 module Trace
   class ZipkinLoggerTracer < ZipkinTracerBase
+    TRACING_KEY = 'Tracing information'
 
     def initialize(options)
       @logger = options[:logger]
+      @logger_accepts_data = @logger.class.name.split('::').first == 'Lorekeeper'
```

Comment:
looks brittle and nowhere in the readme or changelog does it mention lorekeeper. 
also, the ruby way of doing this would be to use `respond_to?` to check the presence of the method you want to use.